pages/popover/placement-test.page.tsx

```diff
@@ -64,14 +65,20 @@ export default function () {
               >
                 <Popover
                   id={`popover-${row}-${col}`}
-                  size="medium"
+                  size="large"
+                  fixedWidth={true}
                   position={position}
                   header="Info"
                   content={<Content />}
                   dismissAriaLabel="Close"
                   triggerType="custom"
                 >
-                  <button id="popover-trigger">Open popover</button>
+                  <button
+                    id="popover-trigger"
+                    style={triggerHeight ? { display: 'block', height: `${triggerHeight}px` } : undefined}
+                  >
+                    {triggerHeight === undefined && 'Open popover'}
```

Comment:
To make sure it is narrow enough that it has more space to the sides than to the top or bottom, and to test that it will expand to the top or bottom despite that. The text is OK when simulating a normal inline button, but for these cases it makes the testing conditions less deterministic.

Reply:
Sounds like one more reason to introduce separate „trigger variant“ param to configure this page